interface/forms/eye_mag/php/taskman_functions.php

```diff
@@ -36,52 +38,59 @@ function make_task($ajax_req)
 
     $query      = "SELECT * FROM users WHERE id=?";
     $to_data    =  sqlQuery($query, array($to_id));
-    $filename   = "Fax_" . $encounter . "_" . $to_data['lname'] . ".pdf";
+    $filename   = "Fax_".$enc."_".$to_data['lname'].".pdf";
 
-    $query = "SELECT * FROM documents where encounter_id=? and foreign_id=? and url like ? and deleted = 0";
-    $doc = sqlQuery($query, array($encounter,$pid,'%' . $filename . '%' ));
+    $query = "SELECT * FROM documents where encounter_id=? and foreign_id=? and url like ?";
```

Comment:
should add back the `and deleted = 0` part to this query or else will collect deleted documents